modules/outbound-domain-event-handler/src/application/index.ts

```diff
@@ -0,0 +1,65 @@
+
+
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the "License") and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list (alphabetical ordering) of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+ * Modusbox
+ - Shashikant Hirugade <shashikant.hirugade@modusbox.com>
+ - Vijay Kumar Guthi <vijaya.guthi@modusbox.com>
+ --------------
+ ******/
+
+'use strict';
+
+import { DefaultLogger } from '@mojaloop/logging-bc-client-lib';
+import { ILogger, LogLevel } from '@mojaloop/logging-bc-public-types-lib';
+
+import { IRunHandler, BC_CONFIG } from '@mojaloop/sdk-scheme-adapter-private-shared-lib';
+import { OutboundEventHandler } from './handler';
+import Config from '../shared/config';
+
+(async () => {
+    // Instantiate logger
+    const logger: ILogger = new DefaultLogger(BC_CONFIG.bcName, 'domain-event-handler', '0.0.1', <LogLevel>Config.get('LOG_LEVEL'));
+
+    // start outboundEventHandler
+    const outboundEventHandler: IRunHandler = new OutboundEventHandler();
+    try {
```

Comment:
We are also missing an API-Service for this handler to provide `health` check end-point, and we could also use this API for "Get Requests" in future.

The `health` check end-point could be a future story, and does not necessarily need to be part of this PR, but IMO it will be necessary for Production deployments.

Reply:
Creating story for this